pkg/solver/server.go

```diff
@@ -51,33 +51,49 @@ func NewSolverServer(
 }
 
 /*
- *
- *
- *
+*
+*
+*
 
- Routes
+# Routes
 
- *
- *
- *
+*
+*
+*
 */
-
 func (solverServer *solverServer) ListenAndServe(ctx context.Context, cm *system.CleanupManager, tracerProvider *trace.TracerProvider) error {
 	router := mux.NewRouter()
 
 	subrouter := router.PathPrefix(http.API_SUB_PATH).Subrouter()
 
 	subrouter.Use(http.CorsMiddleware)
 	subrouter.Use(otelmux.Middleware("solver", otelmux.WithTracerProvider(tracerProvider)))
+
+	exemptIPs := solverServer.options.RateLimiter.ExemptedIPs
+
+	log.Debug().Strs("exemptIPs", exemptIPs).Msg("Loaded rate limit exemptions")
```

Comment:
nit: Maybe move this below where we apply the rate limiting (around line 90)? It would confirm for us that we have applied the exempted IPs.